src/services/projects.js

```diff
@@ -383,8 +390,21 @@ module.exports = class ProjectsHelper {
 					}
 				}
 
+				if (fieldData && Object.keys(entityType.validations).includes('max_length')) {
```

Comment:
@adithyadinesh0412 add validation for note also
The note to the reviewer will have a character limit of 200 characters.
There is a character limit counter at the bottom right corner of the input box. 
This character limit is configurable at the org and the instance level.